Misc/NEWS.d/next/Library/2019-02-10-00-00-13.bpo-35500.1HOMmo.rst

```diff
@@ -0,0 +1 @@
+When mock.assert_called_with is called, the error message that is generated currently does not possess a newline between the AssertionError and the Expected field. Now, the message "expected call not found." will follow the AssertionError. In the next lines, the Expected and Actual values will be displayed.
```

Comment:
Thanks for adding a NEWS entry!

This text would be a great description for the PR, but is too verbose for a NEWS entry. A NEWS entry should be a concise description of the change. In this case, I suggest something along the lines of:

```suggestion
Write expected and actual call parameters on separate lines in mock.assert_called_wtih() assertion errors.  Contributed by Susan Su.
```